Modelica/StateGraph.mo

```diff
@@ -2972,7 +2956,6 @@ standard library (with the new block connectors) and in the
 </html>"));
 end Temporary;
 annotation (
-  obsolete = "Obsolete package due to experimental design",
```

Comment:
This package should not be used anyway even if it no longer uses its own components. So you should keep this annotation or better remove the Temporary package itself.

Reply:
This is to silence Dymola 2019 on the [ReleaseChecks](https://github.com/modelica-tools/msl-release):

> Top class Modelica.StateGraph.Examples.FirstExample specified an obsolete type: Obsolete package due to experimental design.